semantic-ui/src/main/scala/react/semanticui/package.scala

```diff
@@ -41,6 +41,9 @@ package semanticui {
     import elements.placeholder.{PlaceholderLine => SUIPlaceholderLine}
     import elements.placeholder.{PlaceholderHeader => SUIPlaceholderHeader}
 
+    implicit def tagOf2AsC[T, N <: TopNode](tagOf: T)(implicit ev: T => TagOf[N]): js.UndefOr[AsC] =
```

Comment:
How is that it works only after moving it?

Reply:
It seems Scala 3 doesn't automatically imports implicit conversions defined at the package level of used types. It would still work in Scala 3 if we did `import react.semanticui._`.

Moving the implicit conversion to `object As` (which is one of the types of the `as` parameter) makes the compiler pick it up without the need for further `import`s.